crates/fuel-core/src/combined_database.rs

```diff
@@ -79,24 +81,49 @@ impl CombinedDatabase {
     #[cfg(feature = "rocksdb")]
     pub fn open(
         path: &std::path::Path,
-        capacity: usize,
         state_rewind_policy: StateRewindPolicy,
-        max_fds: i32,
+        database_config: DatabaseConfig,
     ) -> crate::database::Result<Self> {
         // Split the fds in equitable manner between the databases
-        let max_fds = match max_fds {
+
+        let max_fds = match database_config.max_fds {
             -1 => -1,
-            _ => max_fds.saturating_div(4),
+            _ => database_config.max_fds.saturating_div(4),
```

Comment:
this can be addressed in a follow up pr, but as per my understanding, we shouldn't distribute it equally, but based on the number of columns per db we have :)

Reply:
Opened an issue : https://github.com/FuelLabs/fuel-core/issues/2531